tests/src/performance/linkbench/scripts/build.cmd

```diff
@@ -0,0 +1,116 @@
+@echo off
+
+REM Usage: Build.cmd <LinkBench assets directory>
+setlocal
+call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\VsDevCmd.bat"
```

Comment:
If this will run in the lab, can we make this assumption about the VS installation location?